### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by zareami10** [cursorless-dev/cursorless#512] [General]
I just wanted to thank you for this great piece of software, not many people might be using it at this moment compared to some other software, but for those like me who do, it makes a huge difference.
Keep up the great work!

**Comment by pokey** [comment]
You're welcome! 😊

**Comment by pokey** [comment]
...I'll also take this opportunity to mention that I've just started accepting [donations](https://github.com/sponsors/pokey) to continue working full-time on Cursorless 😊.  Absolutely no obligation, but if you like what you see today, you'll love [what's next](https://github.com/pokey/cursorless-vscode/wiki/What's-next) for Cursorless

Thanks again for the kind words!

**Reply by zareami10** [reply]
I was wondering  whether it would it be possible to make donations outside of github, for example via librepay?

**Reply by pokey** [reply]
Indeed! https://liberapay.com/pokey/
